ui/packages/app/web/src/components/Targets/LabelsCell.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useState} from 'react';
+import {ChevronDoubleDownIcon, ChevronDoubleUpIcon} from '@heroicons/react/solid';
+import Pill from '../ui/Pill';
+
+export interface ITargetEndpoint {}
+
+const LabelsCell = ({key, labels, discoveredLabels}) => {
+  const [areDiscoveredLabelsVisible, setAreDiscoveredLabelsVisible] = useState<boolean>(false);
+
+  return (
+    <td key={key} className="px-6 py-4 whitespace-nowrap text-sm text-gray-500 flex flex-col w-96">
+      <div className="flex flex-wrap">
+        {labels.length &&
+          labels.map(item => {
+            return (
+              <div className="pb-1 pr-1">
+                <Pill key={item['name']} variant="info">{`${item.name}="${item.value}"`}</Pill>
```

Comment:
save Pill variant as a separate enum instead of using a string as the value (i.e. PillVariant.Info instead of "info" )